packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -234,6 +243,9 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       if (newEl) {
         this.focusZone.focusElement(newEl);
       }
+    } else if (!this.okToAddMore()) {
+      (items[items.length - 1] as IPickerItemProps<T>).selected = true;
```

Comment:
hmm.. not so sure we would get into this branch if items.length is 0 unless itemLimit is set to 0, and then I'm not sure the set focus would be triggered.. 
Will right some tests and correct accordingly..

What if we log warning when usrer sets itemLimit to 0 and recommend using disabled instead? And default to 1 if it is set to 0?